kernel/rros/idle.rs

```diff
@@ -94,7 +94,7 @@ fn __rros_set_idle_schedparam(
     // let mut thread_lock = thread_unwrap.lock();
     let p_unwrap = p.unwrap();
     thread_unwrap.lock().state &= !T_WEAK;
-    let prio = unsafe { (*p_unwrap.locked_data().get()).idle.prio };
+    let prio = unsafe { (*p_unwrap.lock().deref()).idle.prio };
```

Comment:
cc @shannmu

Reply:
I don't know why there is a `locked_data` rather than a `lock`. I think the `lock` is better here.

Reply:
Indeed, the version now has only lock.